spec/lib/tacos_demo.py

```diff
@@ -213,25 +213,15 @@ def parse_comments(
 def parse_comment(
     job_filter: str | None, slices_subpath: Path, comment: str
 ) -> Generator[tuple[str, Slice, str]]:
-    remainder = comment
-    while True:
-        comment, tag_start, remainder = remainder.partition(COMMENT_TAG)
-        if not tag_start:
-            return
-
-        tag, tag_end, remainder = remainder.partition(COMMENT_TAG_END)
-
-        tag = Parse(tag)
-        job = tag.before.first("(")  # )
+    lines = comment.splitlines(keepends=True)
+    points = [i for i, l in enumerate(lines) if COMMENT_TAG in l]
+    points.append(len(lines))
+    chunks = ["".join(lines[i:j]) for i, j in pairwise(points)]
+    for chunk in chunks:
+        before, _tag_start, after = chunk.partition(COMMENT_TAG)
+        job, _tag_end, after = after.partition(COMMENT_TAG_END)
```

Comment:
doesn't the comment_tag always appear at the start and the comment_tag_end appear at the end? this `partition` function seems to imply it's in the middle

Reply:
it appears in the middle of the first line of a chunk. the goal of these two lines is to get the bit between the start and end tags.